config/index.js

```diff
@@ -334,6 +334,32 @@ var conf = convict({
       env: 'SUPPORTED_LANGS'
     }
   },
+  redis: {
+    enabled: {
+      default: true,
```

Comment:
@udaraweerasinghege should we set this to false in dev.json?  OR I wonder if we need to bring back redis to fxa-local-dev as a require dependency, just so all devs test this functionality